xml/System.Text/Decoder.xml

```diff
@@ -171,14 +171,28 @@
         <Parameter Name="completed" Type="System.Boolean" RefType="out" Index="5" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="bytes">To be added.</param>
-        <param name="chars">To be added.</param>
-        <param name="flush">To be added.</param>
-        <param name="bytesUsed">To be added.</param>
-        <param name="charsUsed">To be added.</param>
-        <param name="completed">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="bytes">A read-only span of bytes containing the sequence to convert.</param>
+        <param name="chars">The span to store the converted characters.</param>
+        <param name="flush">
+          <see langword="true" /> to indicate no further data is to be converted; otherwise, <see langword="false" />.</param>
+        <param name="bytesUsed">When this method returns, contains the number of bytes that were produced by the conversion. This parameter is passed uninitialized.</param>
+        <param name="charsUsed">When this method returns, contains the number of characters from <paramref name="chars" /> that were used in the conversion. This parameter is passed uninitialized.</param>
+        <param name="completed">When this method returns, contains <see langword="true" /> if all the characters specified by <paramref name="byteCount" /> were converted; otherwise, <see langword="false" />. This parameter is passed uninitialized.</param>
```

Comment:
the text here and in other parts of this overload still referenced things from the other overloads such as `byteCount`, so please review my edits.